charts/umbrella/README.md

```diff
@@ -467,16 +469,34 @@ helm install -f values-adopter-portal.yaml umbrella . --namespace umbrella --cre
 
 #### Data exchange
 
-:warning: Please be aware of [Note for R24.05](#note-for-r2405)
-
 Involved components:
 
-EDC, MIW, DTR, Vault (data provider and consumer in tx-data-provider), CentralIdP.
+> DBRS-Server-Memory, Data Consumer, Data Provider, SSI Dim Wallet Stub.
```

Comment:
could we please put the list of components at the end, so after line `You can follow the [Curl Steps](../../docs/api/curl/UmbrellaConectorDataExchange.md)`?